src/Microsoft.Azure.SignalR.AspNet/HubHost/SignalRMessageParser.cs

```diff
@@ -60,7 +63,11 @@ public IEnumerable<AppMessage> GetMessages(Message message)
                             var groupName = command.Value;
                             var connectionId = GetName(message.Key, PrefixHelper.ConnectionIdPrefix);
                             var joinGroupWithAckMessage = new JoinGroupWithAckMessage(connectionId, groupName).WithTracingId();
-                            
+                            if (joinGroupWithAckMessage.TracingId != null)
+                            { 
+                                AzureSignalRLog.StartToAddConnectionToGroup(_logger, joinGroupWithAckMessage);
```

Comment:
start doing sth....

Reply:
looks like both `start to do` and `start doing` are valid https://ell.stackexchange.com/questions/105862/started-to-do-and-started-doing